### PR TITLE
fzf

### PR DESCRIPTION
**Pull request by cheecheeo**
Basically the same as #8248.

cc: @magnetophon @matthiasbeyer

### CODE REVIEW

**Comment by magnetophon** [comment]
@cheecheeo thanks a lot for fixing my PR!
I was afk for a few weeks...

**Comment by matthiasbeyer** [comment]
:+1: , also for the commits, nicely done!

File: pkgs/tools/misc/fzf/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  version = "0.9.13";
+  goPackagePath = "github.com/junegunn/fzf";
+  src = fetchFromGitHub {
+    owner = "junegunn";
+    repo = "fzf";
+    rev = "${version}";
+    sha256 = "1m9zbf02d6i47c33jys9lr0krqfjk2dr8jzpfhnrb266qcdb27xi";
+  };
+
+  buildInputs = with goPackages; [ crypto ginkgo gomega junegunn.go-runewidth go-shellwords ncurses syncthing text ];
```
Comment (pSub) [inline pkgs/tools/misc/fzf/default.nix]: Mabye break that line. Looks good otherwise.

File: pkgs/tools/misc/fzf/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  meta = with stdenv.lib; {
```
Comment (pSub) [inline pkgs/tools/misc/fzf/default.nix]: Do you want to add yourself as the maintainer.

**Comment by cheecheeo** [comment]
@magnetophon is it ok with you if I add you back as the maintainer? This is morally your commit, but I didn't want to add you as a maintainer without your permission.

**Comment by magnetophon** [comment]
@cheecheeo 
Feel free to put you, me or both of us as the maintainer.
I'm fine with all of those.

**Comment by cheecheeo** [comment]
@magnetophon added as maintainer, any other comments?

**Comment by junegunn** [comment]
Hey guys, you might want to go for [0.10.0](https://github.com/junegunn/fzf/releases/tag/0.10.0).

**Comment by cheecheeo** [comment]
@junegunn Done. Thanks for the comment.

**Comment by lethalman** [comment]
Travis passed thanks.

**Comment by cheecheeo** [comment]
@lethalman thanks!

**Pull request merged**